commands/command_clean.go

```diff
@@ -44,6 +44,10 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 		file.Close()
 	}
 
+	if err != nil {
+		Panic(err, "Error cleaning asset.")
```

Comment:
Thanks, I noticed that as well so I deleted my pull request. Should have looked more in detail after my rebase of this change. I will be more careful next time. Thank you for a great tool. I really like git-lfs!